app/views/devise/two_factor_authentication/max_login_attempts_reached.html.slim

```diff
@@ -1,6 +1,7 @@
 - title t('upaya.titles.account_locked')
 
-
-h2 = t('upaya.titles.account_locked')
+h1.heading = t('upaya.titles.account_locked')
 p
-  = raw(t('devise.two_factor_authentication.max_login_attempts_reached', time_remaining: user_decorator.lockout_time_remaining_in_words))
+  'Your account is temporarily locked because you have entered the
+  'one-time passcode incorrectly too many times.
+p Please try again in #{user_decorator.lockout_time_remaining_in_words}.
```

Comment:
I still think this should remain localized. If the problem was using HTML within the text, I recommend splitting the two sentences into two different keys in the localization file. The second sentence could be mapped to a key called `please_try_again` for example. This would allow both sentences to be styled separately or combined in the same paragraph if desired.